### DISCUSSION TITLE
Activating both display on tent mode?

### DISCUSSION THREAD

**Discussion by cmlosariaGIS** [Archfx/duo-de#44] [Ideas]
Maybe I am in a minority here but can this be done, allowing the user to activate both display on tent mode?

**Comment by Archfx** [comment]
Yes, it is possible, display compositions are handled through [PostureProcessor](https://github.com/Archfx/duoPosture). You can find the composition modes under [here](https://github.com/Archfx/duoPosture/blob/main/src/main/kotlin/com/thain/duo/PostureProcessorService.kt). You have to add a new configuration for your desired mode.
